drivers/include/mrf24j40.h

```diff
@@ -0,0 +1,455 @@
+/*
+ * Copyright (C) 2016 Hochschule f�r Angewandte Wissenschaften Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    drivers_mrf24j40 MRF24J40 based drivers
+ * @ingroup     drivers_netdev
+ *
+ * This module contains drivers for radio devices in Microchip MRF24J40 series.
+ * The driver is aimed to work with all devices of this series.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for MRF24J40 based drivers
+ *
+ * @author      neo@nenaco.de
+ * @author      koen@bergzand.net
+ */
+
+#ifndef MRF24J40_H_
+#define MRF24J40_H_
+
+#include <stdint.h>
+
+#include "board.h"
+#include "periph/spi.h"
+#include "periph/gpio.h"
+#include "net/netdev2.h"
+#include "net/netdev2/ieee802154.h"
+#include "net/gnrc/nettype.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Maximum possible packet size in byte
+ */
+#define MRF24J40_MAX_PKT_LENGTH        (IEEE802154_FRAME_LEN_MAX)
+
+/**
+ * @brief   Default addresses used if the CPUID module is not present
+ * @{
+ */
+#define MRF24J40_DEFAULT_ADDR_SHORT    (0x0230)
+#define MRF24J40_DEFAULT_ADDR_LONG     (0x1222334455667788)
+/** @} */
```

Comment:
Please use the newly (to be) introduced `uuid` module for address generation -> #6314